fvwm/fvwm3.c

```diff
@@ -1403,7 +1401,8 @@ static void SetRCDefaults(void)
 		ecc.type = Restarting ? EXCT_RESTART : EXCT_INIT;
 		ecc.w.wcontext = C_ROOT;
 		exc = exc_create_context(&ecc, ECC_TYPE | ECC_WCONTEXT);
-		xasprintf(&cmd, defaults[i][0], defaults[i][1], defaults[i][2]);
+		xasprintf(&cmd, "%s%s%s", defaults[i][0], defaults[i][1],
+			defaults[i][2]);
 		execute_function(NULL, exc, cmd, 0);
```

Comment:
This change needs to be in its own commit.  I still want to see:

```
xasprintf(&cmd, "%s %s %s", defaults[i][0], defaults[i][1], defaults[i][2]);
```

So that the onus isn't on the string-handling to inject spaces -- if a string has extra spaces here, it doesn't matter.

Reply:
Extra spaces do matter because the translations are inside strings sent to fvwm for menu items and menu titles inside of quotes, so the extra spaces are not ignored. This will turn `+ "Restart fvwm" Restart` to `+ " Restart fvwm " Restart` which to me makes the menu far harder to control its look with extra spaces (why I removed it).